### DISCUSSION TITLE
Consistent MySQL 'time out' error in Budibase Cloud

### DISCUSSION THREAD

**Discussion by AndyGUK** [Budibase/budibase#4945] [Help] [answered]
The other day I posted here regarding an issue with Budibase cloud that I've been experiencing: https://github.com/Budibase/budibase/discussions/4937

I've since run a new self-hosted instance of Budibase and managed to connect to the MySQL database without issue. In fact, after importing the app, everything seems fine- except for the fact that our cloud instance isn't working.

What's the best way to resolve this issue? Is there some form of support team that will be able to look into the issue or do I simply need to close down the account and create a new one?

**Comment by mjashanks** [comment, marked as answer]
Looking into this now @AndyGUK  - i can see the same issue.

**Reply by AndyGUK** [reply]
Great! Thanks.

**Reply by mjashanks** [reply]
Thanks for finding this. Can you please whitelist the IP `34.246.43.190`.

Will update our documentation.

**Reply by mjashanks** [reply]
see my edit please

**Reply by AndyGUK** [reply]
Great- that seems to work. Thanks!

**Reply by mjashanks** [reply]
Great - sorry for that :)
